plugins-compat-tester-gae/src/main/webapp/index.html

```diff
@@ -286,7 +286,7 @@
                     </table>
                     {{if buildLogPath != ""}}
                     <a name="${pluginName}_${pluginVersion}_${core}" />
-                    <a href="#${pluginName}_${pluginVersion}_${core}" onclick="javascript:displayLog('${pluginName} v${pluginVersion} against ${core}', '${buildLogPath}');"><img class="terminal" src="https://github.com/jenkinsci/jenkins/raw/master/war/src/main/webapp/images/24x24/terminal.png" /></a>
+                     <a href="#${pluginName}_${pluginVersion}_${core}" onclick="javascript:displayLog('${pluginName} v${pluginVersion} against ${core}', '${buildLogPath}');"><img class="terminal" src="https://github.com/jenkinsci/jenkins/raw/master/war/src/main/webapp/images/24x24/terminal.png" /></a>
```

Comment:
Don't add files to a PR in which nothing is changed. https://github.com/jenkinsci/plugin-compat-tester/pull/21/files?w=1 doesn't have `index.html`